lib/jekyll/site.rb

```diff
@@ -67,13 +67,8 @@ def config=(config)
     #
     # Returns nothing.
     def process
-      reset
-      read
-      generate
-      render
-      cleanup
-      write
-      print_stats if config["profile"]
+      return profile_process if config["profile"]
+      process_methods.each { |method| public_send(method) }
```

Comment:
I'm not a big fan of impacting readability of this method like this. This has historically been one of the most crucial parts to understanding Jekyll's build procedure, I don't think we should change it in this drastic of a manner.